src/shared/hooks/useAnalytics.ts

```diff
@@ -36,12 +28,13 @@ function useAnalytics() {
 
       console.info(`Track page view to: ${msg ?? path}`)
 
-      trackPageView({ href: msg ?? path })
+      PageViews.trackPageView(msg ?? path)
     },
-    [prevLocation, trackPageView]
+    [prevLocation]
   )
 
-  return { createPiwikInstance, trackPageVisit }
+  return { trackPageVisit }
 }
 
+export { piwikSiteId, hasPiwikSiteId, createPiwikInstance }
```

Comment:
Why do we need to export these values? Especially the `piwikSiteId`? I can't think of a reason another part of the code would need this

Reply:
Good point. Removed.